packages/flutter/lib/src/cupertino/desktop_text_selection_toolbar.dart

```diff
@@ -0,0 +1,153 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/gestures.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter/widgets.dart';
+
+// Minimal padding from all edges of the selection toolbar to all edges of the
+// screen.
+const double _kToolbarScreenPadding = 8.0;
+
+// These values were measured from a screenshot of TextEdit on MacOS 10.15.7 on
+// a Macbook Pro.
+const double _kToolbarWidth = 222.0;
+const Color _kToolbarBorderColor = Color(0xFF505152);
+const Radius _kToolbarBorderRadius = Radius.circular(4.0);
+const Color _kToolbarBackgroundColor = Color(0xFF2D2E31);
```

Comment:
How do I get this from the theme so that it depends on light/dark mode?

Reply:
CupertinoDynamicColor values in CupertinoColors resolve to different color values, depending on the theme's CupertinoTheme's brightness. CupertinoDynamicColor.resolve(CupertinoColors.systemBlue, context).